addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -434,6 +434,40 @@ def create_employee_from_applicant(self):
         if employee:
             dict_act_window['res_id'] = employee.id
         dict_act_window['view_mode'] = 'form,tree'
+        if len(self) == 1: #only if we create one employee: print rainbow
+            #case of fast recruitment (priority 3)
+            if  (datetime.strptime(self.create_date, '%Y-%m-%d %H:%M:%S') + timedelta(days=7)).date() > date.today():
+                dict_act_window['effect'] = {
+                        'fadeout': 'slow',
+                        'img_url': '/web/static/src/img/smile.svg',
+                        'message': "Only %d days from the application to the recruitment!" %((date.today() - datetime.strptime(self.create_date, '%Y-%m-%d %H:%M:%S').date()).days),
+                        'type': "rainbow_man"
+                }
+            #case of reached recruitment goal for job position (priority 2)
+            if self.job_id and self.job_id.no_of_recruitment:
+                date_start_month = fields.Date.to_string(date(date.today().year, date.today().month, 1))
+                if len(self.search(['&', ('employee_name', '!=', False), ('job_id', '=', self.job_id.id), ('create_date', '>=', date_start_month)])) == self.job_id.no_of_recruitment:
+                    dict_act_window['effect'] = {
+                            'fadeout': 'slow',
+                            'img_url': '/web/static/src/img/smile.svg',
+                            'message': "Great! %d out of %d! You've reached your goal for this job position" %(self.job_id.no_of_recruitment, self.job_id.no_of_recruitment),
+                            'type': "rainbow_man"
+                    }
+            #case of personal record per month (priority 2)
+            applicants = self.env['hr.applicant'].read_group(['&', ('create_uid.id', '=', self.env.uid), ('employee_name', '!=', False)], ['create_date'], ['create_date:month'], lazy=False)
```

Comment:
What are you trying to do  exactly, do you think it is really necessary ? I fear computing that kind of statistics costs a lot of unnecessary queries for nothing.